src/agent/src/windows_service.hpp

```diff
@@ -1,18 +1,45 @@
 #pragma once
 
-#include <string>
+#include <windows_api_facade.hpp>
+
 #define WIN32_LEAN_AND_MEAN
```

Comment:
What is this for?

Reply:
This is for reduce the size of the Win32 header files by excluding some of the less frequently used APIs. Otherwise winsock.h is included generating conflicts with the sockets used by boost.